MekHQ/src/mekhq/campaign/personnel/death/AbstractDeath.java

```diff
@@ -64,7 +64,9 @@ protected AbstractDeath(final RandomDeathMethod method, final CampaignOptions op
         setUseRandomPrisonerDeath(options.isUseRandomPrisonerDeath());
         this.enableRandomDeathSuicideCause = options.isUseRandomDeathSuicideCause();
         this.causes = new HashMap<>();
-        initializeCauses();
+        if (!method.isNone()) {
```

Comment:
Possible to add an opposite version of this for easier reading?

Reply:
I personally find it to be harder to read, which is why I've coded it this way.